examples/cluster/plot_segmentation_toy.py

```diff
@@ -80,8 +81,9 @@
 plt.matshow(img)
 plt.matshow(label_im)
 
-# #############################################################################
-# 2 circles
+# %%
+# 2 Circles
+# ---------
 img = circle1 + circle2
```

Comment:
This heading can be altered similarly to the previous one and a brief section description can be appended as follows:
```suggestion
# Plotting two circles
# --------------------
# Here we repeat the above process but only consider the first two circles
# we generated. Note that this results in a cleaner separation between the
# circles as the region sizes are easier to balance in this case.

img = circle1 + circle2
```
Note that as the internal comments are not present/necessary here, this can be left as a single cell.